azure-samples/samples.json

```diff
@@ -414,6 +420,18 @@
       "githubPath": "Azure-Samples/app-service-java-manage-storage-connections-for-web-apps",
       "description": "Getting started on managing storage connections for Web Apps using Java"
     },
+    {
+      "owner": "jianghaolu",
+      "filePath": "azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageWebAppCosmosDbByMsi.java",
+      "githubPath": "Azure-Samples/app-service-java-access-key-vault-by-msi-for-web-apps",
+      "description": "Getting started on accessing Key Vault with Managed Service Identity for Web Apps using Java"
+    },
+    {
+      "owner": "jianghaolu",
+      "filePath": "azure-samples/src/main/java/com/microsoft/azure/management/appservice/samples/ManageWebAppCosmosDbByMsi.java",
```

Comment:
This should point to a different Java file - ManageWebAppCosmosDbThroughKeyVault.java